FAQ.md

```diff
@@ -288,3 +288,7 @@ By default, APISIX only listens on port 9080 when handling HTTP requests. If you
     ```
 
 2. Reload or restart APISIX
+
+## How could I only enable passive health check?
```

Comment:
How about adding this tip standalone?

Reply:
What exactly does that mean？

Reply:
ping@juzhiyuan ,standalone mean?

Reply:
![image](https://user-images.githubusercontent.com/2106987/99328434-08ff3000-28b7-11eb-9671-6c52c924a933.png)

I mean this PR's content is more like a new QA? How about creating a new QA in the FAQ instead of adding content under `How to change the log level?`

I'm not sure if it's a new QA or the improvements for `How to change the log level?`

Reply:
this PR is a new QA, but it's not under `How to change the log level?`
in my markdown display, it looks like this
![image](https://user-images.githubusercontent.com/30819887/99331181-cd666500-28ba-11eb-9c5c-cd19cee1fc68.png)
just like a new QA

Reply:
Oh I know what happened, looks good to me.